lib/fizzy/parser_expr.cpp

```diff
@@ -564,6 +566,13 @@ parser_result<Code> parse_expr(
         {
             uint32_t imm;
             std::tie(imm, pos) = leb128u_decode<uint32_t>(pos, end);
+
+            if (instr == Instr::local_get || instr == Instr::local_set)
```

Comment:
Need to add `local_tee` here. However it seems that doesn't change the number of successful validation tests, which suggests they are not covered  upstream.